tag.sh

```diff
@@ -1,5 +1,29 @@
 #!/bin/bash
 
+#
+# Build the change log in html.
+#
+# @param $1
+#   Release version.
+# @param $2
+#   Next stable release.
+# @param $3
+#   Filename for the changelog.
+#
+function build_changelog() {
+mapfile -t log < <(git log --format='<li><a href=%x22https://git.drupalcode.org/project/drupal/commit/%H%x22>%s</a></li>%n' "$1"^..."$2")
```

Comment:
`mapfile` is not available with the Darwin version of bash, nor is it available from homebrew. So we would need an alternate strategy or at least checking to see if the command exists first.